index.html

```diff
@@ -49,13 +49,15 @@
           <h3>What does Earth sound like from space?</h3>
         </div>
 
-        <div class="icon-bar">
-          <span class="icon-cols"> Like </span>
-          <span class="icon-cols"> Dislike </span>
-          <span class="icon-cols"> Save </span>
-          <span class="icon-cols"> Share </span>
-          <span class="icon-cols"> More </span>
-        </div>
```

Comment:
Replace span tags with list as you have done in previous review

Reply:
- I believe the same suggestions stated in my first comments also go for this review 🌸

1. Using polite suggestions ✔️ 
2. Stating reasons why a requested change should be implemented ✔️ 
3. Adding emojis to make your review comments friendly✔️ 
4. Ask the student what they think. ✔️ 


_Why don't we edit your comments to meet all the requirements I have just stated above??_ 🤔